taskfiles/main.yml

```diff
@@ -15,6 +15,7 @@ vars:
   BUILD_TYPE: '{{default "release" .BUILD_TYPE}}'
   BUILD_DIR: '{{.BUILD_ROOT}}'
   RP_INSTALL_DIR: '{{.BUILD_DIR}}'
+  DEPOT_TOOLS_DIR: '{{.BUILD_ROOT}}/depot_tools'
```

Comment:
this is used in one taskfile (`taskfiles/dev.yml`), but redefined in another one (`taskfiles/rp.yml`). Since `DEPOT_TOOLS_DIR` directly depends on the value of `BUILD_ROOT`, I'd vote for removing it and instead use `{{.BUILD_ROOT}}/depot_tools` when needed, or define a task-level variable instead as it is done in `taskfiles/dev.yml`